cw-relayer/cmd/price-feeder.go

```diff
@@ -0,0 +1,186 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client/input"
+
+	"github.com/rs/zerolog"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ojo-network/cw-relayer/config"
+	"github.com/ojo-network/cw-relayer/oracle"
+	oracleclient "github.com/ojo-network/cw-relayer/oracle/client"
+)
+
+const (
+	logLevelJSON = "json"
+	logLevelText = "text"
+
+	flagLogLevel  = "log-level"
+	flagLogFormat = "log-format"
+
+	envVariablePass = "CW_RELAYER_PASS"
+	accPrefix       = "wasm"
```

Comment:
is this the appropriate prefix for the 'wasmd' chain?

Reply:
yes, but the acc prefix would change depending on the chain where contracts are deployed